web/profiles/custom/os2loop/modules/os2loop_messages/os2loop_messages.info.yml

```diff
@@ -3,3 +3,5 @@ type: module
 description: 'Create and show messages when content is created or changed'
 core_version_requirement: ^9
 package: 'OS2Loop'
+dependencies:
+  - drupal:os2loop_settings
```

Comment:
Should also depend on `os2loop_subscriptions` since we're using configuration from that module.

Reply:
Added